docs/concepts/personal-access-token.mdx

```diff
@@ -37,8 +37,10 @@ Follow these steps to create API Keys for your Project:
 
 1. Go to <ConsoleLink route="project.developers" />.
 2. Click **Create new API key**
-3. Enter the API Key name and click **Create**
-4. Copy the created API Key
+3. Enter the API Key name
+4. Select an **expiration** date. Note that it is not recommeded to create API keys without an expiry date.
```

Comment:
I disagree with this recommendation. Expiring API keys are a recipe for outages 😆 . Also typo in `recommended`

Reply:
Typo fixed by @vinckr :D.

Do you have an alternative suggestion regarding our wording here? Long-living access tokens are also a security threat, but too short-lived keys are a threat for expiring too soon, creating outages.

Reply:
Maybe just describe it like you did. 

Finding the best time to live for an access token is always a tradeoff between security and usability. For maximum security use a short lived access token, but consider that expiring API keys could lead to outages or other issues. For maximum usability use a long-lived access token, but consider that it can be a security risk if the token leaks. 

(personally I dont understand why 1 month/1 day access token have different security profile - if it leaks you are screwed either way no?)